### PR TITLE
paraview: add wrapQtAppsHook

### PR DESCRIPTION
**Pull request by wucke13**
Also reformatted meta and arguments

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by wucke13** [comment]
@GrahamcOfBorg build paraview
@GrahamcOfBorg eval paraview

File: pkgs/applications/graphics/paraview/default.nix [hunk 1]
```diff
@@ -38,6 +35,7 @@ mkDerivation rec {
   nativeBuildInputs = [
     cmake
     makeWrapper
+    wrapQtAppsHook
```
Comment (worldofpeace) [inline pkgs/applications/graphics/paraview/default.nix]: This already uses the specific qt deriver `mkDerivation` which automatically adds this, plus it's not in the docs to do this explicitly. Can you drop this?

**Review by worldofpeace — CHANGES_REQUESTED** [review]
The actual issue why this isn't wrapped is because it is already in `postInstall`.
What you should do is replace `wrapProgram` with `wrapQtApp`.
And move that to `preFixup`, the phasing is important.

File: pkgs/applications/graphics/paraview/default.nix (hunk 1 above)
Comment (wucke13) [reply pkgs/applications/graphics/paraview/default.nix]: Sure!

**Comment by wucke13** [comment]
@worldofpeace Thank you for educating me! I think I adjusted accordingly. Let's see:
@GrahamcOfBorg build paraview

**Pull request merged**